.circleci/config.yml

```diff
@@ -168,6 +168,24 @@ jobs:
       - store-artifacts:
           path: build/x230-hotp-verification
 
+      - run:
+          name: x230-nkstorecli
+          command: |
+            rm -rf build/x230-nkstorecli/* build/log/* && make --load 2 \
+                V=1 \
+                BOARD=x230-nkstorecli || (find ./build/ -name "*.log" -type f -mmin -1|while read log; do echo ""; echo '==>' "$log" '<=='; echo ""; cat $log;done; exit 1) \
```

Comment:
@daringer Please modify to something similar to 
```
rm -rf build/x230-nkstorecli/* build/log/* && make CPUS=4 \
                V=1 \
                BOARD=x230-nkstorecli || (find ./build/ -name "*.log" -type f -mmin -1|while read log; do echo ""; echo '==>' "$log" '<=='; echo ""; cat $log;done; exit 1) \
```

Since we understood that the way to deal with this in CircleCI was to modify CPUS statement management from Makefile to build env to mitigate CircleCI failing, load statement was not enough to constrain number of threads spawned and resulted in builds failing.

Reply:
just the `CPUS=4`, right? 
also rebased to master...